gluten-core/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -656,6 +656,23 @@ object ExpressionConverter extends SQLConfHelper with Logging {
           Seq(replaceWithExpressionTransformerInternal(c.child, attributeSeq, expressionsMap)),
           c
         )
+      case _: MapFromArrays if BackendsApiManager.getBackendName == "velox" =>
+        // for velox backend, we should mapping map_from_arrays based on the map key dedup policy
+        val mapKeyDedupPolicy = SQLConf.get.getConf(SQLConf.MAP_KEY_DEDUP_POLICY)
+        val newSubstraitExprName =
+          if (mapKeyDedupPolicy == SQLConf.MapKeyDedupPolicy.EXCEPTION.toString) {
+            "map_from_arrays_exception_on_duplicates"
```

Comment:
Is this function name really used?

Reply:
yes, it's related to https://github.com/facebookincubator/velox/pull/10280.

But I think it's more reasonable to check the `mapKeyDedupPolicy` config in velox instead of register two functions. WDYT @PHILO-HE @rui-mo @jinchengchenghh

Reply:
Using a config looks better to me. But it requires some extra code change for that presto function. You can check with Meta guys.